### PR TITLE
Re-do docker image

### PR DESCRIPTION
**Pull request by romcheg**
Build a production-ready image by installing ralph-core
package to vanila ubuntu:bionic base image.

Co-authored-by: Maciej Skorczewski <maciej.skorczewski@allegro.pl>
Co-authored-by: Roman Prykhodchenko <me@romcheg.me>

### CODE REVIEW

File: docker/Dockerfile-prod
Comment (szymi-) [inline docker/Dockerfile-prod]: Email is not needed. Who is or will be the owner of ralph.com domain anyway?

File: docker/provision/init-ralph.sh
Comment (szymi-) [inline docker/provision/init-ralph.sh]: How about

```suggestion
isValidCommand () {
  VALID_COMMANDS=(init upgrade)
  match="$1"
  for i in ${VALID_COMMANDS[@]}; do [[ "$i" == "$match" ]] && return 0 ; done
  return 1
}

if ! isValidCommand $i ; then exit 1 ; fi
```

File: docker/Dockerfile-prod
Comment (romcheg) [reply docker/Dockerfile-prod]: I have removed these values from the dockerfile and introduces them as default values in the script. This way users will be able to set up default user on initialization.

**Review by vi4m — DISMISSED** [review]
Well, it looks ok, but it needs at least second review, since it's big TBH

**Review by szymi- — DISMISSED** [review]

File: docker/docker-compose.yml
Comment (ar4s) [inline docker/docker-compose.yml]: It's equivalent to: 
```
- db
- redis
- nginx
```

File: docker/provision/init-ralph.sh
Comment (ar4s) [inline docker/provision/init-ralph.sh:21]: Where is ``upgrade`` command?

File: docker/provision/start-ralph.sh
Comment (ar4s) [inline docker/provision/start-ralph.sh]: Can we use ``wait-for-it.sh`` for that?

File: docker/ralph.conf.nginx
Comment (ar4s) [inline docker/ralph.conf.nginx]: add
```
expires 1y; # unmutable files
add_header Pragma public;
add_header Cache-Control "public";
access_log off; # optional
```

**Review by mkurek — DISMISSED** [review]

File: docker/ralph.conf.nginx
Comment (ar4s) [inline docker/ralph.conf.nginx]: ```access_log off;```

File: docker/docker-compose.yml
Comment (romcheg) [reply docker/docker-compose.yml]: Done

File: docker/provision/init-ralph.sh
Comment (romcheg) [reply docker/provision/init-ralph.sh:21]: It requires no additional actions atm.

File: docker/provision/start-ralph.sh
Comment (romcheg) [reply docker/provision/start-ralph.sh]: Good question. Sounds like something we should do. Let me test that.

File: docker/ralph.conf.nginx
Comment (romcheg) [reply docker/ralph.conf.nginx]: Done

File: docker/ralph.conf.nginx
Comment (romcheg) [reply docker/ralph.conf.nginx]: Done

**Review by Blejwi — DISMISSED** [review]

File: docker/provision/start-ralph.sh
Comment (romcheg) [reply docker/provision/start-ralph.sh]: Seems to work

**Review by ar4s — APPROVED** [review]

**Review by mkurek — APPROVED** [review]

**Pull request merged**
